tools/ci_build/github/linux/docker/Dockerfile.ubuntu_gpu

```diff
@@ -2,10 +2,11 @@ FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04
 
 ARG PYTHON_VERSION=3.6
 ARG BUILD_EXTR_PAR
+ARG DISTRIBUTED
```

Comment:
can this name (DISTRIBUTED) be more descriptive?

Reply:
I can make it a little more descriptive. Maybe ```DISTRIBUTED_SETUP```

Reply:
I called it ```INSTALL_DEPS_DISTRIBUTED_SETUP```

Reply:
sorry, this comment applies to other DISTRIBUTED's too. but on second thought, maybe here something like INSTALL_DEPS_EXTRA_ARGS would be appropriate and INSTALL_DEPS_DISTRIBUTED_SETUP would be a good name elsewhere

Reply:
Ok, I'll make those changes.